proxy/http/HttpSM.cc

```diff
@@ -5182,7 +5182,7 @@ HttpSM::do_http_server_open(bool raw)
     opt.f_tcp_fastopen = (t_state.txn_conf->sock_option_flag_out & NetVCOptions::SOCK_OPT_TCP_FAST_OPEN);
   }
 
-  opt.ssl_client_cert_name        = t_state.txn_conf->ssl_client_cert_filename;
+  opt.set_ssl_client_cert_name(t_state.txn_conf->ssl_client_cert_filename);
   opt.ssl_client_private_key_name = t_state.txn_conf->ssl_client_private_key_filename;
```

Comment:
These `ssl_client_private_key_name` and `ssl_client_ca_cert_name` has the same issue potentially? Because they're `const char *` and setting pointers in the `t_state.txn_conf` here.

Reply:
NVM, they are not used in `ServerSessionPool` like `ssl_client_cert_name` for now. We can leave them as is.